pom.xml

```diff
@@ -32,18 +32,24 @@
         <changelist>-SNAPSHOT</changelist>
         <gitHubRepo>jenkinsci/${project.artifactId}-plugin</gitHubRepo>
         <java.level>8</java.level>
```

Comment:
Per the release notes of one of the recent plugin POM releases (I cannot recall which) this is a deprecated field and should be deleted. In fact it should be printing a warning in your build:

```suggestion
```

Reply:
Would a missing value cause builds with binary layouts of newer Java version? One default for the JVM on build machine (e.g. Java 17 that might produce something not compatible with Java 11?)

I thought of changing this after bumping required core to current LTS that deprecated Java 8 support, so there would be a modernized release of the plugin usable on older installations.

Reply:
The plugin POM version you are using does not consume this value at all, so it is equivalent to an unused variable. A missing value would not change any results.

Reply:
ACK, updated then